test/FluentMigrator.Tests/Integration/IntegrationTestsSetup.cs

```diff
@@ -0,0 +1,42 @@
+#region License
+//
+// Copyright (c) 2007-2024, Fluent Migrator Project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#endregion
+
+using System.Threading.Tasks;
+
+using FluentMigrator.Tests.Containers;
+
+using NUnit.Framework;
+
+namespace FluentMigrator.Tests.Integration
+{
+    [SetUpFixture]
+    public class IntegrationTestsSetup
+    {
+        [OneTimeSetUp]
+        public async Task Setup()
+        {
+            await Task.WhenAll([
```

Comment:
Can you measure how much memory is consumed by starting all these?
```
long beforeMemory = GC.GetTotalMemory(true); // Forces a GC collection // Code that allocates memory long afterMemory = GC.GetTotalMemory(true); long usedMemory = afterMemory - beforeMemory; Console.WriteLine($"Managed Memory Used: {usedMemory} bytes");
```
Or does testcontainers have a way to monitor the memory of the container?

Reply:
I got negative results, weird
![image](https://github.com/user-attachments/assets/c218c542-16be-4c26-bfff-ddacc5fc9813)

Concerning WithCreateParameterModifier, I found no documentation about it, but seeing the memory usage below, I wouldn't be concerned abgout it for now.